termux-api.c

```diff
@@ -264,17 +272,18 @@ _Noreturn void exec_am_broadcast(int argc, char** argv,
     child_argv[11] = input_address_string;
     child_argv[12] = "--es";
     child_argv[13] = "api_method";
-    child_argv[14] = argv[1];
 
-    // Copy the remaining arguments -2 for first binary and second api name:
-    memcpy(child_argv + extra_args, argv + 2, (argc-1) * sizeof(char*));
+    // Copy all arguments except first one (progname)
+    memcpy(child_argv + extra_args, argv + 1, (argc-1) * sizeof(char*));
 
     // End with NULL:
     child_argv[argc + extra_args - 1] = NULL;
```

Comment:
Now here note that you `argc` is wrong, we didn't copy `argv[0]`, so it shouldn't have been counted in size, and that's why there was as exception before 08bcd733, not because of `0` indexing. If `argc` size used was correct and since we had added an extra space for `NULL`, then `-1` wouldn't have been required.